Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -460,6 +466,15 @@ open class ChatMessageListVC:
             cell.configCell(messageCount: messagesCont)
             cell.transform = .mirrorY
             return cell
+        } else if channelType == .announcement {
```

Comment:
@parth-simformsolutions Please add announcement channel at root level in cellForRow